src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java

```diff
@@ -1132,6 +1132,14 @@ public static void pressEscapeKey() {
     logger.info("Press Escape Key");
     Actions act = new Actions(SeleniumDriver.getDriver());
     act.sendKeys(new CharSequence[]{Keys.ESCAPE}).perform();
+  }
 
+  public static void enterKeyInDeDupe(String pluginProperty, String value) {
```

Comment:
nit:
```suggestion
  public static void enterKeyInDedupe(String pluginProperty, String value) {
```
Please change other instances as well

Reply:
Hi Aryan , I didnt get what is expected here.Can you please clarify here.

Reply:
It's just a small change in the casing: Dedupe instead of DeDupe

Reply:
Resolved changed it to enterKeyInDedupe.